Source/Core/Core/BootManager.cpp

```diff
@@ -300,6 +305,9 @@ bool BootCore(const std::string& _rFilename)
         g_wiimote_sources[WIIMOTE_BALANCE_BOARD] = source;
         WiimoteReal::ChangeWiimoteSource(WIIMOTE_BALANCE_BOARD, source);
       }
+
+      IniFile::Section* sysconf_section = game_ini.GetOrCreateSection("Sysconf");
```

Comment:
@JosJuice - agreed.  I'm happy to do whatever but will say that it was confusing seeing "SelectedLanguage" in the code too because I didn't really realize till later that that was referring to the Gamecube language.  If we change the setting, I'd like to change the code references too.